cms/djangoapps/import_from_modulestore/tests/test_tasks.py

```diff
@@ -1,167 +1,73 @@
-            non_existent_usage_ids,
-            str(self.import_event.uuid),
-            self.content_library.learning_package.id,
-            self.user.id,
-            'xblock',
-            override=True,
+        self.usage_keys_string = ['block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_test']
+        self.learning_package_id = 456
+        self.composition_level = 'component'
+        self.override = False
+
+    @patch('cms.djangoapps.import_from_modulestore.tasks.Import.objects.get')
+    def test_import_not_found(self, mock_import_get):
+        """Test behavior when Import doesn't exist."""
+        mock_import_get.side_effect = Import.DoesNotExist
+
+        import_course_to_library_task.delay(
+            self.import_obj.pk, self.usage_keys_string, self.learning_package_id,
+            self.user.pk, self.composition_level, self.override
         )
-        mock_import_client.assert_not_called()
-
-    def test_cannot_import_staged_content_twice(self):
-        """
-        Tests if after importing staged content into the library,
-        the staged content is deleted and cannot be imported again.
-        """
-        chapters_to_import = [self.chapter, self.chapter2]
-        expected_imported_xblocks = [self.problem, self.video]
-        save_legacy_content_to_staged_content_task(self.import_event.uuid)
 
-        self.import_event.refresh_from_db()
-        self.assertEqual(self.import_event.staged_content_for_import.count(), len(chapters_to_import))
-        self.assertEqual(self.import_event.status, ImportStatus.STAGED)
+        mock_import_get.assert_called_once_with(pk=self.import_obj.pk, user_id=self.user.pk)
 
-        import_course_staged_content_to_library_task(
-            [str(self.chapter.location)],
-            str(self.import_event.uuid),
-            self.content_library.learning_package.id,
-            self.user.id,
-            'xblock',
-            override=True,
+    def test_user_task_status_created(self):
+        """Test that UserTaskStatus is created."""
+        import_course_to_library_task.delay(
+            self.import_obj.pk, self.usage_keys_string, self.learning_package_id,
+            self.user.pk, self.composition_level, self.override
         )
 
-        for xblock in expected_imported_xblocks:
-            self._is_imported(self.library, xblock)
-
-        library_learning_package = LearningPackage.objects.get(id=self.library.learning_package_id)
-        self.assertEqual(library_learning_package.content_set.count(), len(expected_imported_xblocks))
-
-        self.import_event.refresh_from_db()
-        self.assertEqual(self.import_event.status, ImportStatus.IMPORTED)
-        self.assertTrue(not self.import_event.staged_content_for_import.exists())
-        self.assertEqual(self.import_event.publishableentityimport_set.count(), len(expected_imported_xblocks))
+        self.import_obj.refresh_from_db()
+        self.assertIsInstance(self.import_obj.status, UserTaskStatus)
```

Comment:
Again, please explain to me where are the functional tests, why did you completely remove them?